docs/src/drivers/pico-ppmc_fr.adoc

```diff
@@ -18,158 +18,164 @@ sur la première carte.
 
 Installation:
 ----
-    loadrt hal_ppmc port_addr=<addr1>[,<addr2>[,<addr3>...]]
+loadrt hal_ppmc port_addr=<addr1>[,<addr2>[,<addr3>...]]
 ----
 
 Le paramètres _port_addr_ indique au pilote quel port parallèle utiliser.
-Par défaut, _<addr1>_ est en 0x0378, _<addr2>_ et les suivantes ne sont pas 
+Par défaut, _<addr1>_ est en 0x0378, _<addr2>_ et les suivantes ne sont pas
```

Comment:
Contrary to those at ends of paragraphs, aren't these trailing spaces needed for when fixed width lines are reconcatenated into paragraph ?

Reply:
Typically the end of a line is a whitespace. Different languages have found different ways to express a continuation on the next line. But for .adoc I am relaxed that these terminal whitespaces are all evil.